handlers/github_api_handler.js

```diff
@@ -0,0 +1,141 @@
+const _ = require('lodash')
```

Comment:
naming conventions for this module are incorrect. This should be named `handlers/github-api-handler.js` Your linter should have picked this up. Please verify `eslint` is properly checking file naming.